thirdeye/thirdeye-frontend/app/pods/components/anomaly-graph/component.js

```diff
@@ -388,7 +388,7 @@ export default Component.extend({
     const showGraphLegend = this.get('showGraphLegend');
     return {
       position: 'inset',
-      show: showGraphLegend
+      show: false
```

Comment:
We should have left `showGraphLegend` here. As it has changed to false. Also we do like the option for user decide to show the legend by specifying the showGraphLegend's value. Now we force it to not show with ` show: false`

Reply:
Also if you don't use `const showGraphLegend = this.get('showGraphLegend');` we should have removed it.